Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -2625,6 +2624,32 @@ def normalize(input_str):
     }
 }
 
+''' HELPER FUNCTION '''
+
+
+def recursive_convert_to_utf8(replace_to_utf):
```

Comment:
eventually you convert str to unicode. So the name should be `recursive_convert_to_unicode`